docs/content/wiring.md

```diff
@@ -3,9 +3,9 @@ title: Using Parameters, Credentials and Outputs
 description: How to wire parameters, credentials and outputs into steps
 ---
 
-# Parameters, Credentials and Outputs in Porter
+# Parameters, Credentials, Outputs, and Images in Porter
```

Comment:
Can you update this to be the title of the article instead in the front matter and remove this duplicate heading?

Reply:
yes indeed. it will require a change. thanks for catching that! 

I'll update the headline in wiring.md too. Good suggestion.